tests/unit/commands/_utils/test_options.py

```diff
@@ -532,15 +532,15 @@ def test_skip_without_hook_package(self):
 
 class TestNextCommandSuggestions(TestCase):
     def test_generate_next_command_recommendation(self):
-        listOfTuples = [("command1", "description1"), ("command2", "description2"), ("command3", "description3")]
+        listOfTuples = [("description1", "command1"), ("description2", "command2"), ("description3", "command3")]
```

Comment:
A better test would be to reproduce what was done in the real code; that would've helped catch the ordering issue.

Reply:
will do!